pkg/utils/validator/clusterresourceoverride.go

```diff
@@ -79,3 +90,20 @@ func validateClusterResourceOverrideResourceLimit(cro fleetv1alpha1.ClusterResou
 	}
 	return errors.NewAggregate(allErr)
 }
+
+// validateClusterResourceOverrideRuleSelector checks if override rule is selecting resource by name.
+func validateClusterResourceOverrideRuleSelector(cro fleetv1alpha1.ClusterResourceOverride) error {
+	allErr := make([]error, 0)
+	for _, rule := range cro.Spec.Policy.OverrideRules {
+		if rule.ClusterSelector == nil {
+			continue
+		}
+		for _, selector := range rule.ClusterSelector.ClusterSelectorTerms {
+			// Check that only label selector is supported
+			if selector.LabelSelector == nil || selector.PropertySelector != nil || selector.PropertySorter != nil {
+				allErr = append(allErr, fmt.Errorf("label selector is only supported for resource selection %+v", selector))
```

Comment:
```suggestion
			if selector.PropertySelector != nil || selector.PropertySorter != nil {
				allErr = append(allErr, fmt.Errorf("invalid clusterSelector %v: only labelSelector is supported", selector))
```

then we check if the selector.LabelSelector is nil or not.

Reply:
we need to call validateClusterSelector func to validate the selector?